app/Services/OrganizationService.php

```
@@ -0,0 +1,589 @@
+    }
+
+    /**
+     * Clear user-related caches
+     */
+    protected function clearUserCache(User $user): void
+    {
+        Cache::forget("user_organizations_{$user->id}");
+
+        // Clear permission caches for all organizations this user belongs to
+        $user->organizations->each(function ($org) use ($user) {
+            $pattern = "user_permissions_{$user->id}_{$org->id}_*";
```

PHPMD (2.15.0)

Avoid unused local variables such as '$pattern'. (undefined)

(UnusedLocalVariable)